_includes/faqs-query-updatedate.html

```diff
@@ -0,0 +1,16 @@
+
+  <div class="api-explorer">
+    <div class="query">
+    </div>
+    <div class="explorer">
+<div class="api-explorer-form">
+  <textarea class="form-control api-explorer-query autosize" rows="1" style="display:none;">https://api.trade.gov/v2/ita_faqs/search?api_key={{ site.apikey }}&update_date=2014-01-01 TO 2014-11-01</textarea>
+  <button class="btn go pull-left" type="button">http://api.trade.gov/v2/ita_faqs/search?update_date=2014-01-01 TO 2014-11-01</button>
+</div>
+<div class="return">
+  <div class="response api-explorer-section">
+    <pre><span class="default"></span></pre>
+  </div>
+</div>
+    </div>
+  </div>
```

Comment:
If you can fix this, I'll be happy to merge. I'm calling it a day now, so if you need to get this out before evening and you're confident your changes are working go ahead and merge.